pages/settings/devices.vue

```diff
@@ -0,0 +1,253 @@
+          id: item.Id || ''
+        });
+
+        this.pushSnackbarMessage({
+          message: this.$t('settings.devices.deleteDeviceSuccess'),
+          color: 'success'
+        });
+
+        this.devices = (await this.$api.devices.getDevices()).data.Items || [];
+      } catch (error) {
+        this.pushSnackbarMessage({
+          message: this.$t('settings.devices.deleteDeviceError'),
+          color: 'error'
+        });
+
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
+    },
+    async deleteAllDevices(): Promise<void> {
+      try {
+        await this.devices?.forEach(async (device) => {
+          if (this.$store.state.deviceProfile.deviceId === device.Id) {
+            return;
+          }
+          await this.$api.devices.deleteDevice({ id: device.Id || '' });
+        });
+
+        this.pushSnackbarMessage({
+          message: this.$t('settings.devices.deleteAllDevicesSuccess'),
+          color: 'success'
+        });
+
+        this.devices = (await this.$api.devices.getDevices()).data.Items || [];
+      } catch (error) {
+        this.pushSnackbarMessage({
+          message: this.$t('settings.devices.deleteAllDevicesError'),
+          color: 'error'
+        });
+
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
+    },
+    setSelectedDevice(device: DeviceInfo): void {
+      this.selectedDevice = device;
+      this.deviceInfoDialog = true;
+    },
+    closeDialog(): void {
+      this.deviceInfoDialog = false;
+      this.selectedDevice = {};
+    },
+    async deleteSelectedDevice(): Promise<void> {
+      try {
+        await this.$api.devices.deleteDevice({
+          id: this.selectedDevice.Id || ''
+        });
+
+        this.pushSnackbarMessage({
+          message: this.$t('settings.devices.deleteDeviceSuccess'),
+          color: 'success'
+        });
+
+        this.selectedDevice = {};
+
+        this.devices = (await this.$api.devices.getDevices()).data.Items || [];
+      } catch (error) {
+        this.pushSnackbarMessage({
+          message: this.$t('deleteDeviceError'),
+          color: 'error'
+        });
+
+        // eslint-disable-next-line no-console
+        console.error(error);
+      }
+      this.deviceInfoDialog = false;
+    },
+    getDeviceIcon({ AppName, Name }: DeviceInfo): string {
+      switch (AppName?.toLowerCase()) {
+        case 'samsung smart tv':
+          return 'mdi-television';
+        case 'xbox one':
+          return 'microsoft-xbox';
+        case 'sony ps4':
+          return 'mdi-sony-playstation';
+        case 'kodi':
+          return 'mdi-kodi';
+        case 'jellyfin android':
+        case 'android tv':
+          return 'mdi-android';
+        case 'jellyfin web':
+        case 'jellyfin web (vue)':
+          switch (Name?.toLowerCase()) {
+            case 'opera':
+            case 'opera tv':
+            case 'opera android':
```

Comment:
What about doing an ``includes('opera')`` for those devices, so we handle also iOS or whatever strange combination?

Reply:
Do we even handle Opera iOS on any of the clients? Afaik, the user agent string is roughly the same one as Safari (since all iOS browsers are Webkit/Safari at the core).

Reply:
That's true but imo it's cleaner, no need for three cases here for instance with what I propose